src/components/Navigation/Buttons/Button.js

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react'
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome'
+import classesCss from "./Buttons.module.scss";
+
+
+/*
+    iconDefault - default image of Button
```

Comment:
Kinda redundant comments, use them on specific cases where it really worth to explain smth complex.